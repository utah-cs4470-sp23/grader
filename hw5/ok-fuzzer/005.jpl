print ""
show sum [ p6h. : { false } ] ( array [ x7_6 : V_._6Oi3_3 ( ) ] 701. )
let s_2 [ dW , y ] = {}
show l88
print ": '"
type M = { float }
print ""
time show { bl1 ( [ d4__ ] , 0 ) }
let { n , { { x02..43[] } } } = .403
print ""
time let IF_ = pj_C1..2av.
print "_}"
print ""
assert false, "+"
