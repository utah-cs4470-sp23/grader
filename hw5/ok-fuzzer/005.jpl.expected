(PrintCmd "")
(ShowCmd
 (SumLoopExpr
  p6h.
  (TupleLiteralExpr (FalseExpr))
  (ArrayLoopExpr x7_6 (CallExpr V_._6Oi3_3) (FloatExpr 701))))
(LetCmd (ArgLValue (ArrayArgument s_2 dW y)) (TupleLiteralExpr))
(ShowCmd (VarExpr l88))
(PrintCmd ": '")
(TypeCmd M (TupleType (FloatType)))
(PrintCmd "")
(TimeCmd
 (ShowCmd
  (TupleLiteralExpr
   (CallExpr bl1 (ArrayLiteralExpr (VarExpr d4__)) (IntExpr 0)))))
(LetCmd
 (TupleLValue
  (ArgLValue (VarArgument n))
  (TupleLValue (TupleLValue (ArgLValue (ArrayArgument x02..43)))))
 (FloatExpr 0))
(PrintCmd "")
(TimeCmd (LetCmd (ArgLValue (VarArgument IF_)) (VarExpr pj_C1..2av.)))
(PrintCmd "_}")
(PrintCmd "")
(AssertCmd (FalseExpr) "+")
